src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Data availability 
+description: An overview of problems and solutions relating to data availability in Ethereum 
+lang: en
+sidebar: true
+---
+
+In Ethereum, a node proposing a new block is required to make the underlying data available to the remaining peer-to-peer nodes on the network. This concept, known as data availability, is fundamental to the security and functionality of every blockchain, including Ethereum. Data availability is so important that an entire field of research studying the so-called “data availability problem” exists. 
```

Comment:
I think we can explain this in simpler terms. The high-level ELI5 I'd expect to see here is that if we can’t reproduce something with the data we have available, from the blockchain’s perspective, it does not exist.

Reply:
+1. I think it is also a nice place to include a "don't trust, verify" reminder, as ultimately this is why we care about data availability.

Reply:
I created a new intro with ideas from both suggestions. Happy to hear your thoughts on it.